flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -60,7 +64,12 @@ function setup_kubernetes_for_linux {
     local crictl_version crictl_archive
     crictl_version="v1.24.2"
     crictl_archive="crictl-$crictl_version-linux-${arch}.tar.gz"
-    wget -nv "https://github.com/kubernetes-sigs/cri-tools/releases/download/${crictl_version}/${crictl_archive}"
+    download_circtl_url="https://github.com/kubernetes-sigs/cri-tools/releases/download/${crictl_version}/${crictl_archive}"
+    if ! retry_times 3 5 "wget -nv ${download_circtl_url}"; then
```

Comment:
can you double-check that wget works properly (i.e. returns an non-zero exit code) if accessing the website fails? ...to make the retry logic work.